taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpResponse
```

Comment:
If you wanted to add type annotation for `render()`, go for it. Otherwise, remove unused imports.

Reply:
Sorry, for `index()`.